src/api/app/views/webui/user/_involvement_and_activity.html.haml

```diff
@@ -3,35 +3,46 @@
   .card-header{ class: contribution_graph? ? 'd-block d-md-none' : 'd-block' }
     %h5 Involved Projects and Packages
 
-  - if contribution_graph?
-    -# Tabs
+  -# Tabs
+  - if contribution_graph? || Flipper.enabled?(:content_moderation, User.session) && policy(Report.new(reportable: displayed_user)).create?
     .d-none.d-md-block
       %ul.card-header.nav.nav-tabs.pt-2.px-3.pb-0.flex-nowrap.collapsible{ role: 'tablist' }
         %li.nav-item
           = link_to('#involved-projects-and-packages', id: 'involved-projects-and-packages-tab', class: 'nav-link text-nowrap',
                     data: { 'bs-toggle': 'tab' }, role: 'tab', aria: { controls: 'involved-projects-and-packages', selected: false }) do
             Involved Projects/Packages
-        %li.nav-item
-          = link_to('#activity', id: 'activity-tab', class: 'nav-link text-nowrap',
-                    data: { 'bs-toggle': 'tab' }, role: 'tab', aria: { controls: 'activity', selected: false }) do
-            Contributions
+        - if contribution_graph?
+          %li.nav-item
+            = link_to('#activity', id: 'activity-tab', class: 'nav-link text-nowrap',
+                      data: { 'bs-toggle': 'tab' }, role: 'tab', aria: { controls: 'activity', selected: false }) do
+              Contributions
+        - if Flipper.enabled?(:content_moderation, User.session)
+          - if policy(Report.new(reportable: displayed_user)).create?
+            %li.nav-item
+              = link_to('#comments', id: 'comments-tab', class: 'nav-link text-nowrap',
+                        data: { 'bs-toggle': 'tab' }, role: 'tab', aria: { controls: 'comments', selected: false }) do
+                Comments
 
   .card-body.p-0#involvement-and-activity
     = render partial: 'webui/user/involvement', locals: { involved_items_service: involved_items_service }
 
     -# Displaying tabs in medium-large viewports when the Contribution Graph is enabled
-    - if contribution_graph?
+    - if contribution_graph? || Flipper.enabled?(:content_moderation, User.session) && policy(Report.new(reportable: displayed_user)).create?
```

Comment:
> I'm not sure I understand the use of `policy(Report.new(reportable: displayed_user)).create?` here. Why does everyone that can create a report (logged in users which haven't submitted a report against that user yet) see the comments?

I see now that this reasoning is flawed... I it would be better that only Moderators could review it, right?